common/src/main/java/com/microsoft/identity/common/internal/result/MsalBrokerResultAdapter.java

```diff
@@ -599,7 +602,43 @@ public Bundle extractInteractiveRequestBundleFromResultBundle(@NonNull final Bun
     }
 
     @NonNull
-    public AcquireTokenResult getAcquireTokenResultFromResultBundle(@NonNull final Bundle resultBundle) throws BaseException {
+    public AuthorizationResult getDeviceCodeFlowAuthResultFromResultBundle(@NonNull final Bundle resultBundle) throws BaseException, ClientException {
+        String serializedDCFAuthResult = resultBundle.getString(AuthenticationConstants.Broker.BROKER_DCF_AUTH_RESULT);
+        if (serializedDCFAuthResult != null) {
+            AuthorizationResult authorizationResult = ObjectMapper.deserializeJsonStringToObject(serializedDCFAuthResult, MicrosoftStsAuthorizationResult.class);
+            return authorizationResult;
+        }
+
+        // DCF not supported
+        BrokerResult brokerResult = brokerResultFromBundle(resultBundle);
+        if (brokerResult.getErrorCode() != null && brokerResult.getErrorCode().equals(ErrorStrings.DEVICE_CODE_FLOW_NOT_SUPPORTED)) {
+            throw new ClientException(ErrorStrings.DEVICE_CODE_FLOW_NOT_SUPPORTED, "deviceCodeFlowAuthRequest() not supported in BrokerMsalController");
```

Comment:
Add a log statement here

Reply:
yes, when enableDCFInBroker is disbaled . I have added a comment here as well as a log statement